sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_file_system_client.hpp

```diff
@@ -177,6 +177,31 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
         bool recursive,
         const ListPathsSinglePageOptions& options = ListPathsSinglePageOptions()) const;
 
+    /**
+     * @brief Gets the permissions for this file system. The permissions indicate whether
+     * file system data may be accessed publicly.
```

Comment:
You sure it's permissions not access policy? I'm not quite clear about the differences between them, just to make sure.

Reply:
This is to align with the description in Blob, where `permissions` are used. If I were to choose, I would prefer `Access Policy` since permission is just a broader term used elsewhere, such as `SetPermissions`, which has nothing to do with access policies.